README.md

```diff
@@ -42,28 +48,32 @@ Currently users with the admin permission can see/dismiss notifications for all
 
 #### Use the included generic notification
 
-    use Pestopancake\LaravelBackpackNotifications\Notifications\DatabaseNotification;
-    
-    $user = backpack_user();
-    $user->notify(new DatabaseNotification(
-        $type = 'info', // info / success / warning / error
-        $message = 'test notification',
-        $messageLong = 'this is a test notification', // optional
-        $href = '/test', // optional, e.g. backpack_url('/example')
-        $hrefText = 'test notification' // optional
-    ));
+```php
+use Pestopancake\LaravelBackpackNotifications\Notifications\DatabaseNotification;
+
+$user = backpack_user();
+$user->notify(new DatabaseNotification(
+    $type = 'info', // info / success / warning / error
+    $message = 'Test Notification '.rand(1, 99999),
+    $messageLong = 'This is a longer message for the test notification.', // optional
```

Comment:
Actually since the notification title does not get shown in the table view, I think the random number is better placed in the long message:

```suggestion
    $message = 'Test Notification',
    $messageLong = 'This is a longer message for the test notification '.rand(1, 99999), // optional
```